configs/routing_helper_v2.template.yaml

```diff
@@ -0,0 +1,42 @@
+{%- macro make_route_internal(cluster, options) %}
+  cluster: {{ cluster }}
+  {%- if 'timeout' in options -%}
+  timeout: {{ options['timeout'] }},
+  {% endif %}
+  retry_policy:
+    retry_on: 5xx
+{%- endmacro %}
+{%- macro make_route(cluster) -%}
+  {{ make_route_internal(cluster, clusters.get(cluster, {})) }}
+{%- endmacro -%}
+{%- macro internal_cluster_definition(service, options) -%}
+  name: {{ service }}
+  connect_timeout: 0.250s
+  type: EDS
+  eds_cluster_config:
+    eds_config:
+      api_config_source:
+        api_type: REST
+        cluster_names:
+        - main_website
```

Comment:
cluster name here is the sds cluster

Reply:
Do i have to add a new static cluster with same name before?

Reply:
not sure what you mean by "before" in this context.

Yes, if a config source cluster name is defined statically, then the cluster itself (sds in this case) needs to be defined statically as well; https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/config_source.proto#core-apiconfigsource